src/App.js

```diff
@@ -1,28 +1,140 @@
-import React, { Component } from 'react';
-import logo from './logo.svg';
+import React, { Component} from 'react';
 import './App.css';
+import {Layout, Header, Navigation, Drawer, Content, Footer,FooterSection, FooterDropDownSection, FooterLinkList} from 'react-mdl';
+import MainRoute from './routes/mainroute';
+import {Link} from 'react-router-dom';
+import {Badge, Icon} from 'react-mdl';
+import BuildFavBadge from './components/complibrary/buildfavbadge';
+import {Tooltip} from 'reactstrap';
 
+var fs = require('fs');
```

Comment:
❗ this is unused, and will import a built-in node module into a component on the web which may have consequences depending on the build